templates/quarkus/http/pom.xml

```diff
@@ -32,7 +32,7 @@
   <dependencies>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-funqy-http</artifactId>
+      <artifactId>quarkus-funqy-knative-events</artifactId>
```

Comment:
Why `knative-events` for HTTP function?

Reply:
It serves root uri "/".

Reply:
`knative-events` also supports HTTP: GET+uriparams and POST+jsonBody.